File: java/Q41.java
import java.util.Scanner;

public class Q41 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("required  result : ");
        int required = sc.nextInt();

        int result = 0;
        do{
            System.out.print("Enter first value : ");
            int a = sc.nextInt();
            System.out.print("Enter second value : ");
            int b = sc.nextInt();
            System.out.print("Enter operator : ");
            char ch = sc.next().charAt(0);
            switch (ch){
                case '+':
                    result = a+b;
                    break;
                case '-' :
                    result = a - b;
                    break;
                case '*':
                    result = a*b;
                    break;
                case '/' :
                    result = a/b;
                    break;
            }
            System.out.println("The value result : "+result);
        }while(result != required);

        System.out.println("you won ");
    }
}

File: Javascript/Q41.js
const prompt = require("prompt-sync")();

const required = +prompt("Enter required number : ");
let a = null;
let b = null;
let operator = null;
let ans = 0;

do {
  a = +prompt("Enter first value : ");
  operator = prompt("Enter operator : ");
  b = +prompt("Enter second value : ");

  switch (operator) {
    case "+":
      ans = a + b;
      break;
    case "-":
      ans = a - b;
      break;
    case "*":
      ans = a * b;
      break;
    case "/":
      ans = a / b;
  }
  if (ans != required) {
    console.log("Wrong input , try again");
  }
} while (ans != required);

console.log("correct ans");
